api/db/clients/clients.go

```diff
@@ -0,0 +1,44 @@
+package clients
+
+import (
+	uuid "github.com/satori/go.uuid"
+	bh "github.com/timshannon/bolthold"
+
+	"github.com/Zenika/MARCEL/api/db/internal/db"
+)
+
+func Get(id string) (*Client, error) {
+	var c = new(Client)
+
+	if err := db.Store.Get(id, c); err != nil {
+		if err == bh.ErrNotFound {
+			return nil, nil
+		}
+		return nil, err
+	}
+	return c, nil
+}
+
+func List() ([]Client, error) {
+	var clients = []Client{}
+
+	return clients, db.Store.Find(&clients, nil)
+}
+
+func Insert(c *Client) error {
+	c.ID = uuid.NewV4().String()
```

Comment:
We don't check that the given client doesn't already have and id before ?

Reply:
No, this is the responsibility of `Insert()` to generate the id